terra.env

```diff
@@ -172,3 +172,15 @@ source "${VSI_COMMON_DIR}/linux/just_files/docker_functions.bsh"
 : ${COMPOSE_PROJECT_NAME=$(docker_compose_sanitize_project_name "${TERRA_CWD}" "${TERRA_USERNAME}")}
 
 : ${COMPOSE_FILE=${TERRA_CWD}/docker-compose-main.yml}
+
+# `pipenv sync` fails when seeded with pip versions 20.3.0 to 20.3.3.
+# The reported error involves mismatched metadata
+#    ERROR: Requested envcontext==2017.10.19.45 ...
+#    has different version in metadata: '2017.10.19'
+# This error appears to be caused by a new pip resolver introduced in pip 20.3
+#    https://github.com/pypa/pip/issues/9203
+# Pinning the seeded pip version to 20.2.4 resolves the issue. This is
+# accomplished via a VIRTUALENV_* environment variable as per:
+#   https://virtualenv.pypa.io/en/latest/cli_interface.html?highlight=environment%20variables#environment-variables
+# This pin can likely be eliminated with pip version 20.3.4
+: ${VIRTUALENV_PIP=20.2.4}
```

Comment:
Without envcontext, is this failing any more?  If not, perhaps leave `: ${VIRTUALENV_PIP=}` with some comments for posterity, but do not pin the pip version.